### ISSUE TITLE
Corber refuses to acknowledge the presence of project-local `ios-deploy`

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [isleofcode/corber#567]
So, without any changes that I can directly relate to this, we're now getting this error on every attempted `corber build`:

```
Corber: ios-deploy was not found. Please download, build and install version 1.9.2
or greater from https://github.com/phonegap/ios-deploy into your path, or do
'npm install -g ios-deploy'
```

Problem is, `ios-deploy` is very definitely in the project:

```
$ npm ls ios-deploy
ide@0.1.16 /Users/ec/Work/ide
└── ios-deploy@1.9.4
```

Installing `ios-deploy` globally *does* work, but is very undesirable for our environment (slash in general.)

Haven't changed Corber, ember-cli, or Node versions; but *have* recently changed the versions of a few supposedly-unrelated plugins to avoid `ember-cli-babel@5` deprecation warnings; thing is, though, dropping back to the old version in git before those changes *still exhibits this bug*. So no idea what's happened.

Anyway, this is now reproducible across machines, and after wiping out and re-cloning / `npm ci`'ing the whole project, so I'm pretty sure it's a meaningful Corber bug. Happy to help narrow down, too!

Corber: `v1.3.2`
ember-cli: `v2.18.2`
Node: `v8.11.4`

**Comment by alexblom** [comment]
If you have a an iOS device plugged in when doing a build, something upstream in Cordova appears to now be trying to install to your device by default. 

For now try to do a build with your device unplugged, and I'll put fixing this top of the list for the new year.

**Comment by alexblom** [comment]
This will be fixed in 1.3.6 - releasing Friday. Thanks for the report!
